### DISCUSSION TITLE
Can akvorado display traffic path ISP?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1503] [Q&A]
Hi @vincentbernat ,

May i know whether the akvorado can display the traffic path ISP? I means if ISP A want go to ISP Z example ISP A ---> ISP C --> ISP D --> ISP Z ..... If can how we can do like that?

Thanks

**Comment by vincentbernat** [comment]
That's DstASPath.

**Reply by kronicklez** [reply]
HI @vincentbernat 

In the demo.yaml script i can see as-path configured. But when i do sankey graph i cannot see traffic path as per below snapshot? Do i misunderstand something? Supposedly i should see the traffic path right? Thanks and appreciate your feedback

- prefixes: 216.58.206.0/24,2a00:1450:4007:807::2000/124
          aspath: 174,1299,15169
          communities: 174:22004,174:21100
        - prefixes: 179.60.192.0/24,2a03:2880:f130:83:face:b00c:0::/112
          aspath: 1299,1299,32934
          communities: 1299:30000,1299:30220

![image](https://github.com/user-attachments/assets/237dcf12-9a8f-4f0d-a695-0d466a4a3076)

**Reply by vincentbernat** [reply]
https://demo.akvorado.net/visualize/N4IgRglgJhBOCmBjALhA9gOwIYBsQC4AzXAZ3gBoQYBbeDE9eggbRAGVZEBBNkSgERLIeABSzIAFiAC6lOlAIgATAAYlAFgC0ARm071AFRUB2fAFYAHOe0A6bWaUAtPiEIQcyeLEUBJDD8IAITQAVwwoLFgATwACAF4YiAxPWGw8SgBzWCwABwkDKJz4RRIsDABreCiXCRDqMoBRcMUMNAB3GrqytmRI5EUANhiJUNgSGKwMtBccCGoIfvxtFUochAA3dBCSES90BSJSChAhPsVVDR09bUMVbXMrM1t7JxdkWDDEcXhNdfUCADMSko70+31+AwI2iUFkoYQWJEUOABYByiIAvkA=

**Reply by kronicklez** [reply]
Hi @vincentbernat ,

Thanks for sharing your snapshot. May i know whether u use same demo-yaml script right? Or u have add/fine tune that script. I try same as u but cannot get that as-path info.

Thanks

**Reply by vincentbernat** [reply]
It's the exact configuration in the GitHub repository. DstASPath is only for destination. If your destination is the internal network, paths are just the destination AS.

**Reply by kronicklez** [reply]
Hi @vincentbernat ,

I test back using the demo-yaml now i can see result same as yours. But i check actual akvorado i deploy in production cannot see the result DstASPath in sankey graph. But in the dashboard "home" i can see some value DstASPath. Supposedly when we select DstASPath on sankey it should display all the result that have DstASPath right for example like this 

![image](https://github.com/user-attachments/assets/25cec86d-20c0-49f8-987d-166cd3c820a9)




Thanks appreciate some feedback

**Reply by kronicklez** [reply]
Hi @vincentbernat ,

It's ok...i think now i'm understand how to read it. Sorry to disturb you.

Thanks

**Reply by hawbirr** [reply]
hi @kronicklez ,
can you please share how you fix that ?
thank you

**Reply by kronicklez** [reply]
> It's the exact configuration in the GitHub repository. DstASPath is only for destination. If your destination is the internal network, paths are just the destination AS.



Hi @vincentbernat ,

Sorry to bring up the old question. I have question, usually the "internal" referring to interface facing internal network or we can say "core" interface right? May i know whether we still need to sampling this interface so we can see DstASPath? I'm checking on my production network i'm still cannot see the result like u show. Is it mandatory to enable BMP in router also to get as u show in the graph example? Appreciate your feedback
